TUTORIAL.ipynb

```diff
@@ -0,0 +1,1090 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataflows Tutorial\n",
+    "\n",
+    "This tutorial is built as a Jupyter notebook which allows you to run and modify the code inline and can be used as a starting point for new Dataflows projects.\n",
+    "\n",
+    "If you want, you can just skip the installation section and follow the tutorial as-is, copy-pasting the relevant code example to your Python interpreter."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation\n",
+    "\n",
+    "Easiest way to get started on any OS is to [Download and install the latest Python 3 Miniconda distribution](https://conda.io/miniconda.html)\n",
+    "\n",
+    "Open a terminal with Miniconda (or Anaconda) and run the following to create a environment:\n",
+    "\n",
+    "```sh\n",
+    "conda create -n dataflows 'python<3.7' jupyter jupyterlab ipython requests\n",
+    "```\n",
+    "\n",
+    "Activate the environment and install some the required dependencies:\n",
+    "\n",
+    "```sh\n",
+    ". activate dataflows\n",
+    "conda install -c conda-forge 'cchardet<2'\n",
```

Comment:
why the <2 limit?

Reply:
see previous comment

Reply:
Let's try with the new tabulator and see if it helps?